### PR TITLE
Feature/tacking config manager

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Added tracking-config service manager

# Impacts
No

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Added packetfence-tracking-config service manager

### CODE REVIEW

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -647,6 +647,16 @@ pfstats=enabled
 # Location of the pfstats binary.
 pfstats_binary=/usr/local/pf/sbin/pfstats
 
+# services.tracking-config
+#
+# Should tracking-config be managed by PacketFence?
+tracking-config=enabled
+#
+# services.tracking_binary
```
Comment (cgx) [inline conf/pf.conf.defaults]: `services.tracking-config_binary`

File: conf/pf.conf.defaults [hunk 2]
```diff
@@ -647,6 +647,16 @@ pfstats=enabled
...
+#
+# Location of the pfstats binary.
```
Comment (cgx) [inline conf/pf.conf.defaults]: `s/pfstats/tracking-config/`

File: conf/pf.conf.defaults [hunk 3]
```diff
@@ -647,6 +647,16 @@ pfstats=enabled
...
+tracking-config_binary=/dev/null
```
Comment (cgx) [inline conf/pf.conf.defaults:658]: /dev/null? Really?

Comment (fdurand) [reply conf/pf.conf.defaults:658]: Yes there is no binary, i can keep like that or add an exception in checkup.pm

**Pull request merged**